### DISCUSSION TITLE
Add fingerprint unlock to desktop wallet

### DISCUSSION THREAD

**Discussion by nop33** [alephium/alephium-frontend#305] [Feature requests]
A user on Discord requested:

> It would be awsome being able to unlock alephium's desktop wallet with finger print. Especially on macos, as it is well implemented across the whole OS

Related issue: https://github.com/alephium/alephium-frontend/issues/63

**Comment by diomark** [comment]
windows HELLO is well implemented in Windows as well. (for fingerprint/pin/etc unlocks)
